src/main/kotlin/racingcar/model/Round.kt

```diff
@@ -0,0 +1,15 @@
+package racingcar.model
+
+class Round(private val numberGenerator: NumberGenerator) {
+    fun play(cars: List<Car>) {
+        cars.forEach { car ->
+            if (numberGenerator.generate() >= MIN_FORWARD_VALUE) {
+                car.moveForward()
+            }
```

Comment:
이번 프리코스 코드리뷰 과정을 통해 가지게된 생각을 공유해봅니다.
저도 인협님처럼 바로 메소드를 사용했었는데, 한번 변수를 거친 후 사용하는게 디버깅 측면이나, 명확성 측면에서 좀더 유리하겠다는 생각을 가졌습니다!😀
```suggestion
            val randomNumber = numberGenerator.generate()
            if (randomNumber >= MIN_FORWARD_VALUE) {
                car.moveForward()
            }
```